static/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -118,11 +118,12 @@ type SpanBarProps = {
   toggleEmbeddedChildren:
     | ((props: {eventSlug: string; orgSlug: string}) => void)
     | undefined;
-  toggleSiblingSpanGroup: ((span: SpanType) => void) | undefined;
+  toggleSiblingSpanGroup: ((span: SpanType, occurrence: number) => void) | undefined;
   toggleSpanGroup: (() => void) | undefined;
   toggleSpanTree: () => void;
   trace: Readonly<ParsedTraceType>;
   treeDepth: number;
+  groupOccurrence?: number;
```

Comment:
I chose to name it `groupOccurrence` here as opposed to just `occurrence`, since I thought it would be confusing without clarification, as this prop is part of just a regular `spanBar`